packages/trpc/server/routers/loggedInViewer/deleteMeWithoutPassword.handler.ts

```diff
@@ -12,7 +12,7 @@ type DeleteMeWithoutPasswordOptions = {
 };
 
 export const deleteMeWithoutPasswordHandler = async ({ ctx }: DeleteMeWithoutPasswordOptions) => {
-  const user = await prisma.user.findUnique({
+  const user = await prisma.user.findFirst({
```

Comment:
```suggestion
  const user = await prisma.user.findUnique({
```

Reply:
@joeauyeung This and some other comments are marked resolved but there aren't any changes. Can you add a comment if it's not supposed to change before resolving the comment.
If you have made the suggested change you can just resolve the conversation.

Reply:
With the comments without any changes, it looks like it synced properly with `main` so there wouldn't be any changes reflected in this PR.